iphone/Classes/TiUISearchBar.m

```diff
@@ -54,6 +54,19 @@ -(UISearchBar*)searchBar
 	return searchView;
 }	
 
+-(void)setSearchBar:(UISearchBar *)searchBar
+{
+    if (searchView)
+    {
+        [searchView removeFromSuperview];
+        [searchView setDelegate:nil];
+        RELEASE_TO_NIL(searchView);
+    }
+    searchView = searchBar;
+    [searchView setDelegate:self];
+    [self addSubview:searchView];
+}
```

Comment:
Not sure what's this used for. It's used in https://github.com/appcelerator/titanium_mobile/pull/9066/files#diff-c25f31a8654c7459f7b97be688ab44efR2210 but using a setter seems to be the wrong way. I'm curious if we should o that way, especially because it will add a new sub-view every time it's called. So if the user changes the search-bar over time (using `searchView` / `searchBar`, multiple search-bars will just be added above each other.

Also brackets miss-formatted again, that needs to be checked in general.